decentralised-identity.md

```diff
@@ -0,0 +1,138 @@
+
+This approach aligns with the idea that your digital identity should serve you, not the other way around. It's all about convenience and empowerment. You don't have to remember a dozen passwords or deal with complicated sign-up processes. Your digital identity is yours, tailored to fit your needs and preferences. It's a shift from the traditional model where companies hold the reins – now, it's you calling the shots.
+
+Convenience, the double edge sword:
+
+Convenience in technology is a double-edged sword, offering unparalleled ease alongside potential risks. While innovations like biometric authentication, one-click payments, and streamlined user experiences enhance convenience, they also introduce concerns such as data security and privacy. The more seamless and interconnected our digital experiences become, the more we rely on storing personal information online, making individuals vulnerable to cyber threats and data breaches. Striking the right balance between convenience and security is crucial; prioritizing one over the other can lead to significant drawbacks. Therefore, as technology advances to make our lives more convenient, it is essential to implement robust safeguards, educate users on responsible practices, and continuously adapt security measures to mitigate potential risks.
+
+### Conclusion:
+
+In summary, decentralized identity aligns with the cypherpunk philosophy by promoting user autonomy, privacy, and security. It envisions a future where individuals have more control over their digital identities. Imagine having a digital identity that's flexible, private, and works seamlessly across various online platforms. It's like having a key that fits perfectly into every digital lock you encounter.
+
+Moreover, the interoperability of decentralized identity ensures that your digital persona isn't confined to a single platform – it travels with you. By eliminating single points of failure, the system becomes more resilient and less vulnerable to large-scale breaches.
+
+In a user-centric world, you decide how, when, and where to use your digital identity. This cypherpunk vision pushes us towards a future where individuals hold the reins of their digital lives, supported by secure, private, and user-friendly decentralized identity systems.
+
+However, it's crucial to acknowledge that this utopian vision of DID is not without its challenges. The potential drawbacks include concerns about scalability, energy consumption, and the complex integration of blockchain technology. Achieving widespread adoption and navigating legal and regulatory uncertainties also present ongoing hurdles. 
+
+In conclusion, while decentralized identity holds immense promise, addressing these challenges is imperative to ensure a secure, private, and user-friendly evolution.
+
```

Comment:
Thanks for the edit. It's generally a lot less biased but the article still skims over important details and basic information, here's some basic questions that should be addressed:

* What data needs to be collected by DID and why?
* How is this data going to be used and how could systems like this mitigate the risks?
* Why it's a bad idea to store your internet data with your personal data?
* Should our personal data and identity be kept off the internet / blockchain entirely and why?
* What do protocol providers have to gain from building these platforms and collecting your data?
* What is the potential for backdoor Corp / Gov keys?
* What is the potential for your data to be harvested or sold on these new systems?
* What are the alternatives to storing your data?
* Can a data-less system be created for our internet persona?
* Why is it a bad idea to store personal data on-chain and what are the long term implementations?

Passports, Drivers licences and Birth certificates already exist and are freely accessible by governments and pretty much anyone else that wants to pay for this data. In my view, this needs to be address at a government level to make this data more secure and private. But ultimately, governments holding all of your personal data will never change. So my thought here is, why would you connect your personal data to your internet data? The system I envision keeps these things completely separate. To the point that, if you leaked all of your internet data there would be no trace of your real identity. 

The cypherpunk principles advocate for anonymity, privacy and security. The minute your personal data is linked with your internet data this is gone. Any system asking for personal data should be heavily scrutinised as a potential bad actor.

Reply:
Yes agree with all above. These are the questions that must be asked. Maybe I will include these at the end of the article in a section. I think it is a good article to get people talking. I tried to take the approach that it will at some stage be possible but yes I've clearly not addressed a lot of the downsides. We aren't quite there yet but I do envisage hopefully a place in the future that we can have a DID, can interact without the data being compromised, can control a lot more privacy than having the privacy on the companies side/servers.

I'll add all this in a paragraph at the end of the article